Makefile

```diff
@@ -23,22 +26,42 @@ test-postgresql:
 
 # Engine-specific test environments. In most cases you should run these with
 # `make -e ADDITIONAL_PYTEST_OPTIONS="--use-persistent-source-schema" test-<engine_type>`
+#
+# See: [Persistent Source Schema](/GLOSSARY.md#persistent-source-schema)
 .PHONY: test-bigquery
 test-bigquery:
 	hatch -v run bigquery-env:pytest -vv -n $(PARALLELISM) $(ADDITIONAL_PYTEST_OPTIONS) metricflow/test/
 
+.PHONY: populate-persistent-source-schema-bigquery
+populate-persistent-source-schema-bigquery:
+	hatch -v run bigquery-env:pytest -vv $(ADDITIONAL_PYTEST_OPTIONS) $(POPULATE_PERSISTENT_SOURCE_SCHEMA)
```

Comment:
I think it's better to define a make command that just works, instead of one that relies on someone knowing that they have to set up the `ADDITIONAL_PYTEST_OPTIONS=--use-persistent-source-schema` environment flag.

I just know I'm going to perpetually be in a loop of:

`make populate-persistent....databricks`
<fail>
`make -e "ADDITIONAL_PYTEST_OPTIONS=--use-persistent-source-schema" populate-persistent...databricks`

Reply:
That's fair. Updated.